src/components/title-search-list.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import QueryList from './query-list';
+import TitleListItem from './title-list-item';
+
+export default function TitleSearchList({
+  location,
+  params,
+  collection,
+  fetch,
+  onPage
+}) {
+  return (
+    <QueryList
+      type="titles"
+      page={parseInt(params.page || 1, 10)}
+      fetch={fetch}
+      collection={collection}
+      onPage={onPage}
+      itemHeight={80}
```

Comment:
magic number?

Reply:
Yea, we could wait until mount and get the height of an individual item. But if that one item we choose to measure gets a weird layout, that would result in all of the offset and scroll calculations being off.

I saw one scrolling component that didn't require an `itemHeight` prop, but it also didn't provide an `onScroll` or an `onUpdate` method so we can get the `offset`. So to calculate that we'd have to have an `itemHeight` in that case anyway.